virtcontainers/network.go

```diff
@@ -143,6 +143,8 @@ type NetworkInterfacePair struct {
 type NetworkConfig struct {
 	NetNSPath         string
 	NetNsCreated      bool
+	UseNetmon         bool
+	NetmonConfig      NetmonConfig
```

Comment:
Just a small question. In some structs, the bool field "enable" is in the config struct. Why they are separate here and in oci/utils.go?

Reply:
Just made the modification as I agree it didn't make any sense to separate this field from the `NetmonConfig` structure.